lib/Net/Kubernetes/Role/ResourceCatalog.pm

```diff
@@ -0,0 +1,76 @@
+package Net::Kubernetes::Role::ResourceCatalog;
+
+use Moose::Role;
+
+requires 'server_version';
+requires 'url';
+
+my %endpoint_catalog = (
+	1.5 => {
```

Comment:
Do we not care about earlier versions any more or does the default behavior work for any prior version?